server/app/views/admin/questions/QuestionConfig.java

```diff
@@ -293,7 +294,10 @@ public DivTag getContainer() {
    */
   private static ImmutableList<SelectWithLabel.OptionValue> stateOptions() {
     return ImmutableList.of(
-        SelectWithLabel.OptionValue.builder().setLabel("-- Leave blank --").setValue("-").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("Washington").setValue("WA").build());
+        SelectWithLabel.OptionValue.builder().setLabel("-- Leave blank --").setValue("").build(),
+        SelectWithLabel.OptionValue.builder()
+            .setLabel("Washington")
+            .setValue("Washington")
```

Comment:
Why did you change value from "WA" to "Washington"? Are you we using full state names in address question?

Reply:
requirement as per- https://github.com/civiform/civiform/issues/3630